pom.xml

```diff
@@ -691,6 +693,19 @@
                 <artifactId>caffeine</artifactId>
                 <version>${caffeine.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.jsoup</groupId>
+                <artifactId>jsoup</artifactId>
+                <version>${jsoup.version}</version>
+            </dependency>
```

Comment:
How about using GitHub API to instead of jsoup to fetch sql text?

Reply:
fixed